app/templates/views/security.html

```diff
@@ -1,89 +1,93 @@
-  <p class="govuk-body">Notify complies with data protection law. To make sure it stays compliant, there are regular legal reviews of the service’s:</p>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>privacy policy</li>
-    <li>terms of use</li>
-    <li>approach to data sharing</li>
-  </ul>
+    <h2 class="heading-medium" id="data">Data</h2>
+    <p class="govuk-body">On Emergency Alerts, data is encrypted when:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>it passes through the service</li>
+      <li>it’s stored on the service</li>
+    </ul>
+    <p class="govuk-body">Any recipient data you upload is only held for 7 days.</p>
+    <p class="govuk-body">If you <a class="govuk-link govuk-link--no-visited-state" href="/features/email#send-files">send a file by email</a>, the file will be available for the recipient to download for up to 78 weeks (18 months).</p>
+    <p class="govuk-body">The Cabinet Office acts as data processor for Emergency Alerts. Your organisation is the data controller.</p>
+    <h3 class="heading-small">Data Protection Act</h3>
+    <p class="govuk-body">Emergency Alerts complies with data protection law. To make sure it stays compliant, there are regular legal reviews of the service’s:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>privacy policy</li>
+      <li>terms of use</li>
+      <li>approach to data sharing</li>
+    </ul>
 
-  <h2 class="heading-medium" id="technical-security">Technical security</h2>
-  <p class="govuk-body">Other technical security controls on Notify include:</p>
-  <ul class="govuk-list govuk-list--bullet">
-    <li>compliance with National Cyber Security Centre (NCSC) Cloud Security Principles</li>
-    <li>protective monitoring to record activity, and raise alerts about any suspicious activity</li>
-    <li>using JSON Web Tokens, to avoid sending API keys when your service talks to Notify</li>
-  </ul>
+    <h2 class="heading-medium" id="technical-security">Technical security</h2>
+    <p class="govuk-body">Other technical security controls on Emergency Alerts include:</p>
+    <ul class="govuk-list govuk-list--bullet">
+      <li>compliance with National Cyber Security Centre (NCSC) Cloud Security Principles</li>
+      <li>protective monitoring to record activity, and raise alerts about any suspicious activity</li>
+      <li>using JSON Web Tokens, to avoid sending API keys when your service talks to Emergency Alerts</li>
+    </ul>
 
-  <h3 class="heading-small">Protect sensitive information</h3>
-  <p class="govuk-body">Some messages include sensitive information like security codes or password reset links.</p>
-  <p class="govuk-body">If you’re sending a message with sensitive information, you can choose to hide those details on the Notify dashboard once the message has been sent. This means that only the message recipient will be able to see that information.</p>
+    <h3 class="heading-small">Protect sensitive information</h3>
```

Comment:
This section (the <h3> and the two following <p>) should probably be removed - users won't be uploading sensitive PII to the system.